tools/src/main/scala/scala/scalanative/build/Discover.scala

```diff
@@ -153,7 +160,8 @@ object Discover {
       case None => {
         val binaryNames = binaryVersions.flatMap {
           case (major, minor) =>
-            Seq(s"$binaryName$major$minor", s"$binaryName-$major.$minor")
+            val sep = if (minor == "") "" else "."
+            Seq(s"$binaryName$major$minor", s"$binaryName-$major${sep}$minor")
```

Comment:
If you want to support `clang8` and `clang-8` being discovered, you should add more potential binary names here instead of dropping support for `clang-8.0` which is for example used on ubuntu.

Reply:
I believe that with the change to the clangVersions data structure in the latest commit
any of clangN, clang-N, or clang-N.0 will be found, if extant, by the code you highlighted , where N  either 7 or 8.

To my eyes, changing the data structure makes the intent of supporting/testing both  "" and "0" as minor
versions clearer than hiding some tricky manipulation in a line of code.

If you like, I can revert the current commit and handle the complexities in the line you indicated.

Reply:
It doesn't hurt to support more potential names here. After all this is auto discovery implementation, it should do its best effort at finding a clang binary name.